docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,349 @@
+    * How long did `Quantum ESPRESSO`_ actually run (wall time)?
+
+From calculations to workflows
+------------------------------
+
+AiiDA can help you run individual calculations but it is really designed to help you run workflows that involve several calculations, while automatically keeping track of the provenance for full reproducibility.
+
+As the final step, we are going to launch the ``PwBandStructure`` workflow of the ``aiida-quantumespresso`` plugin.
+
+.. literalinclude:: include/snippets/demo_bands.py
+
+Download the :download:`demo_bands.py <include/snippets/demo_bands.py>` snippet and run it using
+
+.. code-block:: console
+
+    $ verdi run demo_bands.py
+
+This workflow will:
+
+    #. Determine the primitive cell of the input structure.
+    #. Run a calculation on the primitive cell to relax both the cell and the atomic positions (``vc-relax``).
+    #. Refine the symmetry of the relaxed structure, and find a standardised primitive cell using SeeK-path_.
+    #. Run a self-consistent field calculation on the refined structure.
+    #. Run a band structure calculation at fixed Kohn-Sham potential along a standard path between high-symmetry k-points determined by SeeK-path_.
+
+The workflow uses the PBE exchange-correlation functional with suitable pseudopotentials and energy cutoffs from the `SSSP library version 1.1 <https://www.materialscloud.org/discover/sssp/table/efficiency>`_.
+
+.. K-point mesh is selected to have a minimum k-point density of 0.2 Å-1
+   A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations
+
+The workflow should take ~10 minutes on your virtual machine.
+You may notice that ``verdi process list`` now shows more than one entry:
+
+.. code-block:: console
+
+    $ verdi process list
+      PK  Created    Process label             Process State    Process status
+    ----  ---------  ------------------------  ---------------  --------------------------------------
+     218  6s ago     PwBandStructureWorkChain  ⏵ Waiting        Waiting for child processes: 233
+     233  4s ago     PwBandsWorkChain          ⏵ Waiting        Waiting for child processes: 235
+     235  3s ago     PwRelaxWorkChain          ⏵ Waiting        Waiting for child processes: 238
+     238  2s ago     PwBaseWorkChain           ⏵ Waiting        Waiting for child processes: 244
+     244  1s ago     PwCalculation             ⏵ Waiting        Monitoring scheduler: job state QUEUED
+
+    Total results: 5
+
+    Info: last time an entry changed state: 0s ago (at 12:45:40 on 2020-11-25)
+
+While you wait for the workflow to complete, let's start exploring its provenance.
+
+The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+
+Start the AiiDA REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+
+and open the |provenance browser| (from the browser inside the virtual machine).
```

Comment:
This was marked as resolved but still says `from the browser inside the virtual machine`, which should be changed no? Or am I missing something?

Also, from the conversation with @ltalirz I deduced you already managed to make this work with some sort of redirection and will include this in the setups for their ssh-config? It worked using [my setup](https://github.com/ramirezfranciscof/aiida-core/wiki/Port-Redirection) but I don't know if this was the same way you used and I guess the test should be with that.

Reply:
> This was marked as resolved but still says `from the browser inside the virtual machine`, which should be changed no? Or am I missing something?

Whoops! Totally forgot to update the text after getting the RestAPI to work. Good that you caught it 😅. Will fix it 👍.

> Also, from the conversation with @ltalirz I deduced you already managed to make this work with some sort of redirection and will include this in the setups for their ssh-config? It worked using [my setup](https://github.com/ramirezfranciscof/aiida-core/wiki/Port-Redirection) but I don't know if this was the same way you used and I guess the test should be with that.

Yes, this will (potentially) be discussed in the setup. I still have to test if we can use the RestAPI if we run the tutorial in AiiDAlab.